packages/react-scripts/package.json

```diff
@@ -21,55 +21,55 @@
     "react-scripts": "./bin/react-scripts.js"
   },
   "dependencies": {
-    "@babel/core": "7.0.0-beta.38",
-    "@babel/runtime": "7.0.0-beta.38",
-    "autoprefixer": "7.2.5",
+    "@babel/core": "7.0.0-beta.40",
+    "@babel/runtime": "7.0.0-beta.40",
+    "autoprefixer": "8.0.0",
     "babel-core": "7.0.0-bridge.0",
-    "babel-eslint": "8.2.1",
-    "babel-jest": "22.1.0",
+    "babel-eslint": "8.2.2",
+    "babel-jest": "22.4.1",
     "babel-loader": "8.0.0-beta.0",
     "babel-plugin-named-asset-import": "^0.1.0",
     "babel-preset-react-app": "^3.1.1",
     "case-sensitive-paths-webpack-plugin": "2.1.1",
-    "chalk": "2.3.0",
-    "css-loader": "0.28.9",
+    "chalk": "2.3.1",
+    "css-loader": "0.28.10",
     "dotenv": "5.0.0",
     "dotenv-expand": "4.2.0",
-    "eslint": "4.15.0",
+    "eslint": "4.18.1",
     "eslint-config-react-app": "^2.1.0",
     "eslint-loader": "1.9.0",
-    "eslint-plugin-flowtype": "2.41.0",
-    "eslint-plugin-import": "2.8.0",
+    "eslint-plugin-flowtype": "2.46.1",
+    "eslint-plugin-import": "2.9.0",
     "eslint-plugin-jsx-a11y": "6.0.3",
-    "eslint-plugin-react": "7.5.1",
-    "extract-text-webpack-plugin": "3.0.2",
-    "file-loader": "1.1.6",
+    "eslint-plugin-react": "7.7.0",
+    "extract-text-webpack-plugin": "4.0.0-alpha.0",
+    "file-loader": "1.1.9",
     "fs-extra": "5.0.0",
-    "graphql": "0.12.3",
-    "graphql-tag": "2.6.1",
-    "html-webpack-plugin": "2.30.1",
+    "graphql": "0.13.1",
+    "graphql-tag": "2.8.0",
+    "html-webpack-plugin": "webpack-contrib/html-webpack-plugin#master",
     "identity-obj-proxy": "3.0.0",
-    "jest": "22.1.2",
+    "jest": "22.4.2",
     "object-assign": "4.1.1",
-    "postcss-flexbugs-fixes": "3.2.0",
-    "postcss-loader": "2.0.10",
+    "postcss-flexbugs-fixes": "3.3.0",
+    "postcss-loader": "2.1.1",
     "promise": "8.0.1",
     "raf": "3.4.0",
     "react-dev-utils": "^5.0.0",
-    "style-loader": "0.19.1",
+    "style-loader": "0.20.2",
     "svgr": "1.8.1",
-    "sw-precache-webpack-plugin": "0.11.4",
-    "thread-loader": "1.1.2",
-    "uglifyjs-webpack-plugin": "1.1.6",
+    "sw-precache-webpack-plugin": "goldhand/sw-precache-webpack-plugin#master",
+    "thread-loader": "1.1.4",
+    "uglifyjs-webpack-plugin": "1.2.2",
```

Comment:
i think we should just use the built-in uglifyjs now that we use webpack 4 https://github.com/webpack-contrib/uglifyjs-webpack-plugin#uglifyjs-webpack-plugin

Reply:
I agree but the info on the link is obsolete. (Error: webpack.optimize.UglifyJsPlugin has been removed, please use config.optimization.minimize instead.)   

I will move it to config.optimization when the webpack 4 doc is released.

Reply:
i see, thanks for the info!